Jenkinsfile

```diff
@@ -77,81 +77,71 @@ ansiColor('xterm') {
-             wait
-             '''
-             junit '**/reports/junit/wdio/*.xml'
+              sh '''#!/bin/bash -e
+              source ~/.nvm/nvm.sh &> /dev/null
+              nvm use v8.9.4
+              npx netlify deploy -t ${NETLIFY_TOKEN}
+              '''
             }
           }
 
-          stage('Bump version'){
-            sh '''#!/bin/bash -ex
-            source ~/.nvm/nvm.sh
-            nvm use v8.9.1
-            git diff
-            npm version patch -m "build %s"
-            version=`grep "version" package.json | head -1 | awk -F: '{ print $2 }' | sed 's/[", ]//g'`
-            echo $version > .version
-            '''
-            packageJsonVersion = readFile '.version'
-          }
-
-          stage('Build for CDN'){
+          stage('Run Journey Tests') {
             withCredentials([
-              usernamePassword(credentialsId: 'MESSAGE_DEMO_CLIENT', passwordVariable: 'MESSAGE_DEMO_CLIENT_SECRET', usernameVariable: 'MESSAGE_DEMO_CLIENT_ID'),
-              file(credentialsId: 'web-sdk-cdn-private-key', variable: 'PRIVATE_KEY_PATH'),
-              string(credentialsId: 'web-sdk-cdn-private-key-passphrase', variable: 'PRIVATE_KEY_PASSPHRASE'),
+              string(credentialsId: 'REACT_WIDGETS_CLIENT_ID', variable: 'CISCOSPARK_CLIENT_ID'),
+              string(credentialsId: 'REACT_WIDGETS_CLIENT_SECRET', variable: 'CISCOSPARK_CLIENT_SECRET'),
+              usernamePassword(credentialsId: 'SAUCE_LABS_VALIDATED_MERGE_CREDENTIALS', passwordVariable: 'SAUCE_ACCESS_KEY', usernameVariable: 'SAUCE_USERNAME'),
             ]) {
-              sh '''#!/bin/bash -ex
-              source ~/.nvm/nvm.sh
-              nvm use v8.9.1
-              version=`cat .version`
-              NODE_ENV=production
-              BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${version}/" npm run build:package widget-space
-              BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${version}/" npm run build sri widget-space
-              BUILD_BUNDLE_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${version}/" BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${version}/demo/" npm run build:package widget-space-demo
-              BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/" npm run build:package widget-recents
-              BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/" npm run build sri widget-recents
-              BUILD_BUNDLE_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/" BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${version}/demo/" npm run build:package widget-recents-demo
-              '''
+             sh '''#!/bin/bash -e
+             source ~/.nvm/nvm.sh &> /dev/null
+             nvm use v8.9.4
+             export JOURNEY_TEST_BASE_URL=https://practical-roentgen-7d4de0.netlify.com
```

Comment:
Should we make this a param somewhere or env var so that we can easily change it if necessary?